### PR TITLE
Fixed typo and added license

### LINKED ISSUE #3: License
Could you please define a License for this so I can figure out if I am able to use it?

### LINKED ISSUE #8: darkVibrantSwatch should fall back to vibrantSwatch in Palette:generateEmptySwatches
See [Palette.php#L129](https://github.com/marijnvdwerf/material-palette-php/blob/7c937f0627e9337c7c032ac2d02e3bebcb2403d0/src/Palette.php#L129). Minor typo, but if I'm reading it right, it looks like it'll prevent an empty `darkVibrantSwatch` from being generated from `vibrantSwatch`.

### PR DESCRIPTION
**Pull request by on2**
Resolves #3 and fixes #8.

### CODE REVIEW

File: tests/PaletteTest.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+namespace marijnvdwerf\palette\Tests;
+
+use Intervention\Image\ImageManager;
+use marijnvdwerf\palette\Palette;
+use PHPUnit\Framework\TestCase;
+
+class PaletteTest extends TestCase
+{
+    public function testGenerateEmptySwatches()
```
Comment (marijnvdwerf) [inline tests/PaletteTest.php:11]: I think this test case would be better if it only looked at the histogram -> palette conversion. That we we'd just need to feed it a small list of colours, and only validate that part of the library. Something like  `$pal = new Palette(Swatch[] $swatches)`, and then validate where the proper ones were picked and generated,

**Comment by marijnvdwerf** [comment]
Thanks for adding the test cases! I don't mind the one with the existing images, but I left a comment on how I'd prefer to see the other one.

**Comment by on2** [comment]
I appreciate they are not testing a unit of code but wanted something there to check the changes I made didn't break anything.

File: tests/PaletteTest.php (hunk 1 above)
Comment (on2) [reply tests/PaletteTest.php:11]: Yes, that makes sense. I will try to identify a concise palette where a dark vibrant swatch wouldn't have previously been generated. At the minute I just have a sample set of images missing a swatch and I'm not sure what's consistent with their histograms.

**Comment by marijnvdwerf** [comment]
I merged in most of your changes, and published a new tag (1.3.0)

**Pull request closed without merging**